README.md

```diff
@@ -6,15 +6,40 @@
 
 Open-RMF Web is a collection of packages that provide a web-based interface for users to visualize and control all aspects of Open-RMF deployments.
 
-- [Getting started](#getting-started)
+- [Quick start with docker](#quick-start-with-docker)
+- [Getting started from source](#getting-started-from-source)
 - [API server](packages/api-server)
 - [API client](packages/api-client)
 - [Dashboard](packages/dashboard)
 - [Configuration](#configuration)
 - [Contribution guide](#contribution-guide)
-- [Roadmap](https://github.com/open-rmf/rmf-web/wiki/Open-RMF-Web-Dashboard)
+- [Roadmap](https://github.com/orgs/open-rmf/projects/10)
 
-# Getting started
+# Quick start with docker
```

Comment:
Would be good to put a link to rmf demos.

Reply:
https://github.com/open-rmf/rmf-web/pull/994/commits/31b6489a5c6b69a38900f3fb52dfdff5d6a43eaa